presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -211,6 +336,66 @@ public static double mod(@SqlType(StandardTypes.DOUBLE) double num1, @SqlType(St
         return num1 % num2;
     }
 
+    private static SqlScalarFunction decimalModFunction()
```

Comment:
drop this and reuse definition from `modulusOperator` from `DecimalOperators`.
You cannot directly reuse `DecimalOperators.DECIMAL_MODULUS_OPERATOR` as that uses operator name mangling. But reuse the internal implementation.